titration/utils/ui_state/titration/setup_titration.py

```diff
@@ -72,8 +80,6 @@ def loop(self):
             self.titrator.lcd.print("Yes: 1", line=2)
             self.titrator.lcd.print("No (use old): 0", line=3)
             self.titrator.lcd.print(
-                "{0:>2.3f} pH: {1:>2.4f} V".format(
-                    constants.PH_REF_PH, constants.PH_REF_VOLTAGE
-                ),
+                f"{constants.PH_REF_PH} pH: {constants.PH_REF_VOLTAGE} V",
```

Comment:
This change concerns me.  Why is the number formatting being removed.

Reply:
I would like to discuss this change in person